### DISCUSSION TITLE
[Request] Config: Minimum length for video downloads

### DISCUSSION THREAD

**Discussion by secrets69** [UltimaHoarder/UltimaScraper#1014] [Ideas]
When downloading videos, there are so many models that have like 3-second videos. Most of the time, I only care about the videos that are at least like 20 seconds long.

It would be great to have a param like `"minimum_video_length" : 20` that makes it so the code only downloads videos that are, in this case, 20 seconds or longer.

Same vein, I guess `minimum_audio_length` would make sense as well.

**Comment by UltimaHoarder** [comment]
The API doesn't specify a video length
